File: javascript/1019.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var val = lines.shift();
function convertTime(num) {
    var hour = Math.floor(num / 3600);
    var minutes = Math.round(((Math.floor(num / 60)/60) - Math.floor(Math.floor(num / 60)/60)) * 60);
    var seconds = Math.round((((num-(hour * 3600)) / 60) - minutes) * 60);

    return console.log(hour+':'+minutes+':'+seconds);
}

convertTime(val);

File: python/1019.py
import math
i=int(input())
def ct(num):
    h=math.floor(num/3600)
    m=round(((math.floor(num/60)/60)-math.floor(math.floor(num/60)/60))*60)
    s=round((((num-(h*3600))/60)-m)*60)
    return print(str(h)+':'+str(m)+':'+str(s))
ct(i)